### PR TITLE
remove makefile in favor of package.json scripts; CSR -> CS

### PR DESCRIPTION
**Pull request by EndangeredMassa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,10 +21,10 @@
     "touch": "0.0.3"
   },
   "devDependencies": {
-    "coffee-script-redux": "2.0.0-beta7",
-    "mocha": "~1.12.1"
+    "coffee-script": "^1.9.1"
   },
   "scripts": {
-    "test": "echo 'write some tests'"
+    "build": "./node_modules/.bin/coffee -cbo lib src",
```
Comment (jkrems) [inline package.json]: package.json scripts have node_modules/.bin at the front of their PATH by default - just `coffee` is enough.

**Comment by jkrems** [comment]
:confetti_ball: - LGTM!

File: package.json (hunk 1 above)
Comment (EndangeredMassa) [inline package.json]: Ah, right. I copy-pasted this. Will fix.

**Comment by EndangeredMassa** [comment]
Simplified the build task

**Comment by jkrems** [comment]
:+1: - down with the Makefile!

**Pull request merged**